src/cyw43_lwip.c

```diff
@@ -124,6 +124,31 @@ STATIC err_t cyw43_netif_init(struct netif *netif) {
 #define CYW43_HOST_NAME "PYBD"
 #endif
 
+#if LWIP_DHCP
+// need to zero out to get isconnected() working
+#undef  CYW43_DEFAULT_IP_ADDRESS
+#endif
+
+#ifndef CYW43_DEFAULT_IP_ADDRESS
+#define CYW43_DEFAULT_IP_ADDRESS LWIP_MAKEU32(0, 0, 0, 0)
+#endif
+
+#ifndef CYW43_DEFAULT_IP_MASK
+#define CYW43_DEFAULT_IP_MASK LWIP_MAKEU32(255, 255, 255, 0)
+#endif
+
+#ifndef CYW43_DEFAULT_IP_GATEWAY
+#define CYW43_DEFAULT_IP_GATEWAY LWIP_MAKEU32(192, 168, 0, 1)
+#endif
+
+#ifndef CYW43_DEFAULT_IP_DNS
+#define CYW43_DEFAULT_IP_DNS LWIP_MAKEU32(8, 8, 8, 8)
+#endif
+
+#ifndef CYW43_DEFAULT_AP_IP_ADDRESS
+#define CYW43_DEFAULT_AP_IP_ADDRESS LWIP_MAKEU32(192, 168, 4, 1)
+#endif
```

Comment:
Can you please move these to `cyw43_config.h`, to keep all config in one place?

Reply:
Done